src/itemProcessingUtils.js

```diff
@@ -0,0 +1,69 @@
+function removeInvalidOptions(option) {
+  return Object.keys(option).includes('value', 'contemnt') &&
```

Comment:
This is an interesting typo. It was in the code before so I didn't alter it. I'm pretty sure this code can't be working with the typo, should I change it now?

Reply:
You can change it. `includes` only accepts one argument, so `'contemnt'` can be removed. I'm fairly confident that what this code should be doing is checking that `option` is an object with `value` and `content` keys, and that the `content` key is not `null` or `undefined`.

Reply:
There are at least 3 files with this spelling mistake. I wonder.

Reply:
The other two files are in a different repo/project though (in `vue-form-elements`). And it appears the setup is the same; `includes` only accepts one argument so it's doing nothing in all cases.

Reply:
I could add a second includes call, checking for content (with correct spelling) ...

Reply:
@JCook21 Because of the `option.content != null` check, adding that 2nd includes would be redundant. If there's no `content` key, then `option.content` would be `undefined` and so `option.content != null` would return `false`.

So I think it's enough to only remove `'contemnt'`.